types/expo/index.d.ts

```diff
@@ -1010,150 +1000,479 @@ export namespace Constants {
-    type Addresses = 'addresses';
-    type Image = 'image';
-    type Thumbnail = 'thumbnail';
-    type Note = 'note';
-    type Birthday = 'birthday';
-    type NonGregorianBirthday = 'nonGregorianBirthday';
-    type NamePrefix = 'namePrefix';
-    type NameSuffix = 'nameSuffix';
-    type PhoneticFirstName = 'phoneticFirstName';
-    type PhoneticMiddleName = 'phoneticMiddleName';
-    type PhoneticLastName = 'phoneticLastName';
-    type SocialProfiles = 'socialProfiles';
-    type InstantMessageAddresses = 'instantMessageAddresses';
-    type UrlAddresses = 'urlAddresses';
-    type Dates = 'dates';
-    type Relationships = 'relationships';
-
-    const PHONE_NUMBERS: PhoneNumbers;
-    const EMAILS: Emails;
-    const ADDRESSES: Addresses;
-    const IMAGE: Image;
-    const THUMBNAIL: Thumbnail;
-    const NOTE: Note;
-    const BIRTHDAY: Birthday;
-    const NON_GREGORIAN_BIRTHDAY: NonGregorianBirthday;
-    const NAME_PREFIX: NamePrefix;
-    const NAME_SUFFIX: NameSuffix;
-    const PHONETIC_FIRST_NAME: PhoneticFirstName;
-    const PHONETIC_MIDDLE_NAME: PhoneticMiddleName;
-    const PHONETIC_LAST_NAME: PhoneticLastName;
-    const SOCIAL_PROFILES: SocialProfiles;
-    const IM_ADDRESSES: InstantMessageAddresses;
-    const URLS: UrlAddresses;
-    const DATES: Dates;
-    const RELATIONSHIPS: Relationships;
-
-    type FieldType = PhoneNumbers | Emails | Addresses | Image | Thumbnail |
-        Note | Birthday | NonGregorianBirthday | NamePrefix | NameSuffix |
-        PhoneticFirstName | PhoneticMiddleName | PhoneticLastName | SocialProfiles |
-        InstantMessageAddresses | UrlAddresses | Dates | Relationships;
+    // Constants
+
+    namespace Fields {
+        type ID = 'id';
+        type Name = 'name';
+        type FirstName = 'firstName';
+        type MiddleName = 'middleName';
+        type LastName = 'lastName';
+        type NamePrefix = 'namePrefix';
+        type NameSuffix = 'nameSuffix';
+        type PhoneticFirstName = 'phoneticFirstName';
+        type PhoneticMiddleName = 'phoneticMiddleName';
+        type PhoneticLastName = 'phoneticLastName';
+        type Birthday = 'birthday';
+        type Emails = 'emails';
+        type PhoneNumbers = 'phoneNumbers';
+        type Addresses = 'addresses';
+        type InstantMessageAddresses = 'instantMessageAddresses';
+        type UrlAddresses = 'urlAddresses';
+        type Company = 'company';
+        type JobTitle = 'jobTitle';
+        type Department = 'department';
+        type ImageAvailable = 'imageAvailable';
+        type Image = 'image';
+        type Note = 'note';
+        type Dates = 'dates';
+        type Relationships = 'relationships';
+        type Nickname = 'nickname';
+        type RawImage = 'rawImage';
+        type MaidenName = 'maidenName';
+        type ContactType = 'contactType';
+        type SocialProfiles = 'socialProfiles';
+        type NonGregorianBirthday = 'nonGregorianBirthday';
+
+        const ID: ID;
```

Comment:
I'm not sure why we need to split it:

```
type ID = 'id';
const ID: ID;
```

instead of

```
const ID: 'id'
```

but I see that the older version use this mechanism too..

Reply:
That's why I kept it.